services/web/client/source/resource/qxapp/node-meta-v0.0.1.json

```diff
@@ -0,0 +1,327 @@
+            "pattern": "^(service)/(computational|dynamic)(/[^\\s/]+)+$",
+            "examples": [
+                "service/computational/itis/sleeper",
+                "service/dynamic/itis/3dviewer"
+            ]
+        },
+        "version": {
+            "type": "string",
+            "description": "semantic version number",
+            "pattern": "^(0|[1-9]\\d*)(\\.(0|[1-9]\\d*)){2}(-(0|[1-9]\\d*|\\d*[-a-zA-Z][-\\da-zA-Z]*)(\\.(0|[1-9]\\d*|\\d*[-a-zA-Z][-\\da-zA-Z]*))*)?(\\+[-\\da-zA-Z]+(\\.[-\\da-zA-Z-]+)*)?$",
+            "examples": [
+                "1.0.0",
+                "0.0.1"
+            ]
+        },
+        "type": {
+          "type": "string",
+          "description": "service type",
+          "enum": ["computational","dynamic"],
+          "examples": [
+              "computational"
+          ]
+        },
+        "name": {
+            "type": "string",
+            "description": "short, human readable name for the node",
+            "examples": [
+                "Fast Counter"
+            ]
+        },
+        "description": {
+            "type": "string",
+            "description": "human readable description of the purpose of the node",
+            "examples": [
+                "Our best node type",
+                "The mother of all nodes, makes your numbers shine!"
+            ]
+        },
+        "authors": {
+            "type": "array",
+            "minItems": 1,
+            "items": {
+                "type": "object",
+                "required": ["name", "email"],
+                "additionalProperties": false,
+                "properties": {
+                    "name": {
+                        "type": "string",
+                        "description": "Name of the author",
+                        "examples": [
+                            "Sun Bak",
+                            "Delenn"
+                        ]
+                    },
+                    "email": {
+                        "description": "Email address",
+                        "type": "string",
+                        "format": "email",
+                        "examples": [
+                            "sun@sense.eight",
+                            "deleen@minbar.bab"
+                        ]
+                    },
+                    "affiliation": {
+                        "description": "Affiliation of the author",
+                        "type": "string",
+                        "examples": [
+                            "Sense8",
+                            "Babylon 5"
+                        ]
+                    }
+                }
+            }
+        },
+        "contact": {
+            "type": "string",
+            "format": "email",
+            "description": "email to correspond to the authors about the node",
+            "examples": [
+                "lab@net.flix"
+            ]
+        },
+        "inputs": {
+            "type": "object",
```

Comment:
so if I get it right, now the key is the name of the object, which is automatically unique in a dictionary?
Also from the schema, label, description, type, displayOrder are required fields. They do not appear in some of the fake data...

Reply:
Correct.
Some fake data might be deprecated. I'll update it.

Reply:
Done